build/pipeline.yml

```diff
@@ -0,0 +1,58 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+pr:
+  drafts: false
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: microsoft/vscode-emmet-helper
+      ref: main
+      endpoint: Monaco
+
+parameters:
+  - name: publishPackage
+    displayName: 🚀 Publish emmet-helper
+    type: boolean
+    default: false
+
+extends:
+  template: azure-pipelines/npm-package/pipeline.yml@templates
+  parameters:
+    npmPackages:
+      - name: \@vscode/emmet-helper
+
+        buildPlatforms:
+          - name: Linux
+            nodeVersions:
+              - 16.x
+          - name: MacOS
+            nodeVersions:
+              - 16.x
+          - name: Windows
+            nodeVersions:
+              - 16.x
+
+        buildSteps:
+          - script: yarn --frozen-lockfile
+            displayName: Install dependencies
+
+          - script: yarn compile
+            displayName: Compile npm package
+
+        testSteps:

```

Comment:
`testSteps` are executed after `buildSteps` so you should only keep the `yarn test` script here.